addons/im_livechat/views/im_livechat_channel_templates.xml

```diff
@@ -25,35 +26,19 @@
                     </script>
                     <style type="text/css">
                         body {
-                            height: 100%;
-                            font-size: 16px;
                             font-weight: 400;
                             font-family: "Lato", "Lucida Grande", "Helvetica neue", "Helvetica", "Verdana", "Arial", sans-serif;
-                            overflow: hidden;
-                            overflow-y: auto;
-                            display: block;
                             margin: 0;
                             padding: 0;
-                            border: none;
                             width: 100%;
                             height: 100%;
-                            background: #C9C8E0;
-                            background-image: -webkit-linear-gradient(top, #7c7bad, #ddddee);
-                            background-image: -moz-linear-gradient(top, #7c7bad, #ddddee);
-                            background-image: -ms-linear-gradient(top, #7c7bad, #ddddee);
-                            background-image: -o-linear-gradient(top, #7c7bad, #ddddee);
                             background-image: linear-gradient(to bottom, #7c7bad, #ddddee);
-                            filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#7c7bad', endColorstr='#ddddee',GradientType=0 );
-                            -webkit-background-size: cover;
-                            -moz-background-size: cover;
-                            -o-background-size: cover;
                             background-size: cover;
                             background-repeat: no-repeat;
                             background-attachment: fixed;
                         }
                         .main {
                             position: absolute;
```

Comment:
While cleaning the template css, couldn't we set body to flex/column and align the inner div properly instead of playing with positions/negative margins?